test_cli.py

```diff
@@ -1,16 +1,13 @@
 # -*- coding: utf-8 -*-
 """
 Tests all the CLI functionality end-to-end.
-
```

Comment:
Why are we removing all these newlines?  Seems like it makes it harder to read.

Reply:
Not sure what happened here. I think when I copied the file over from another branch the spaces got removed. I fixed the spaces now.